### DISCUSSION TITLE
What is the best practice to change bootstrap5 default max-width of the containers div to a different size

### DISCUSSION THREAD

**Discussion by yoscbd** [holger1411/understrap#1798] [Q&A] [answered]
Hi all,
I'm building a new design with Figma,
In the design i got the max with is 1440px.

I'm using bootstrap 5 with my understrap theme where the container max width is 1320px:

`@media (min-width: 1400px)
.container, .container-lg, .container-md, .container-sm, .container-xl, .container-xxl {
    max-width: 1320px;
}`

what is the best way to change the max with to 1440?
should i override the "container" class and set a new max width or is there a better way? :)

**Comment by IanDelMar** [comment, marked as answer]
Add this to your [understrap](https://github.com/understrap/understrap)/[src](https://github.com/understrap/understrap/tree/main/src)/[sass](https://github.com/understrap/understrap/tree/main/src/sass)/[theme](https://github.com/understrap/understrap/tree/main/src/sass/theme)/_theme_variables.scss:

```scss
$container-max-widths: (
  sm: 540px,
  md: 720px,
  lg: 960px,
  xl: 1140px,
  xxl: 1400px  // default value: 1320px
);
```

**Reply by yoscbd** [reply]
Thank you, I will! :)
